### PR TITLE
Fix Setup.py to not fail under cygwin/msys2

### PR DESCRIPTION
**Pull request by DragonCrafted87**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -30,6 +30,10 @@
         'zip',
         '2a616cbb5b15aec8238f22c0d62dede1b6d155798adc45ff4d0206395a8a5833',
     ),
+    'cygwin': (
+        'zip',
+        '2a616cbb5b15aec8238f22c0d62dede1b6d155798adc45ff4d0206395a8a5833',
+    ),
 }
```
Comment (asottile) [inline setup.py:37]: perhaps to make this easier to update, do:
```python
POSTFIX_SHA256['cygwin'] = POSTFIX_SHA256['win32']
```

File: setup.py [hunk 2]
```diff
@@ -30,6 +30,10 @@
         'zip',
         '2a616cbb5b15aec8238f22c0d62dede1b6d155798adc45ff4d0206395a8a5833',
     ),
+    'cygwin': (
+        'zip',
```
Comment (asottile) [inline setup.py:34]: asdf

File: setup.py (hunk 1 above)
Comment (DragonCrafted87) [reply setup.py]: good idea i completely missed that last night

**Review by asottile — APPROVED** [review]

**Comment by asottile** [comment]
thanks for the patch! I'm going to wait to release this until #41 is in too since these are in a similar place

**Pull request merged**